OpenChange/MAPIStoreRecurrenceUtils.m

```diff
@@ -65,6 +65,11 @@ - (void) setupRecurrenceWithMasterEntity: (iCalRepeatableEntityObject *) entity
   [entity removeAllExceptionRules];
   [entity removeAllExceptionDates];
 
+  if (rp == NULL) {
+    /* No recurrence pattern to set, so we return after removing the old ones */
+    return;
```

Comment:
If that is case at any time. It requires to have a log line indicating this strange pattern using `[self logWithFormat:]` or `NSLog` as fallback.